API Field limit

f3lan opened the pull request:
1. Added the API field limits.
2. Updated the SDK
3. Removed unused file
4. Formatted the JS File
5. Added line breaks to the source code for better readability.

f3lan committed:
Updated gitignore file

* Why was this change necessary?
Because waf lock files are not used to commit.

* How does it address the problem?
added file type to gitignore
https://github.com/D-32/PebbleSBB/issues/2

f3lan committed:
Removed unused waf file

* Why was this change necessary?
We do not use this file in the repo.
* How does it address the problem?
Removed it.
https://github.com/D-32/PebbleSBB/issues/2

f3lan committed:
Updated to the newest SDK

* Why was this change necessary?
It is always better to develop with the news SDK ;)
Used the pebble build in update script.
* Are there any side effects?
I tested this on my machine, was strait forward.
https://github.com/D-32/PebbleSBB/issues/2

f3lan committed:
Added the field limits to the urls.

* Why was this change necessary?
The pebble app is mutch faster if we do only get this info we really use.
* How does it address the problem?
The public transport api provides a mechanism to limit the fields. This can be
done by using the url parameter fields[].
* Are there any side effects?
I tested it with my hw app and the emulator. It was really faster in loading
the data. I did not see any negativ side effects.

https://github.com/D-32/PebbleSBB/issues/2

D-32 commented:
Nice, thanks man!
Just a small request, could we maybe put each field limit parameter on a new line and append it to the string? Otherwise these urls become very messy...

f3lan committed:
Added Line breaks to the url string.

* Why was this change necessary?
Because the urls looked a bit messy.
* How does it address the problem?
By adding line breaks to the source code.

https://github.com/D-32/PebbleSBB/issues/2

f3lan commented:
@D-32 Of course! I added some line breaks to the source code.

D-32 commented:
Thanks! :)
I guess I'll do some testing & a release some time in the future...
